src/modules/Contribute/pages/service.tsx

```diff
@@ -228,27 +229,29 @@ Thank you very much`;
                 direction="left"
                 small={true}
               >
-                {t('contribute:service_page.back', 'Go back')}
+                {t('back', { ns: 'contribute/service' })}
```

Comment:
Here would be `contribute:service_page.back`

`{t('contribute:service_page.back')}`
instead of 
`{t('back', { ns: 'contribute/service' })}`

Reply:
If we use `{t('contribute:service_page.back')}` it's will write translations in `contribute.json`file and we want to write in `contribute/service.json`